types/src/transaction/transaction.rs

```diff
@@ -492,6 +493,20 @@ impl HeapSizeOf for Transaction {
     }
 }
 
+fn is_prev_out_duplicated(burns: &[AssetTransferInput], inputs: &[AssetTransferInput]) -> Result<(), Error> {
```

Comment:
How about renaming it to `verify_duplicated_prev_out`?
`is_*` seems to return a boolean.

Reply:
Changed to `check_duplication_in_prev_out`.